config.go

```diff
@@ -57,63 +57,60 @@ type ConfigSection interface {
-	Api                APIConfig               `yaml:"api" bson:"api" json:"api" id:"api"`
-	ApiUrl             string                  `yaml:"api_url" bson:"api_url" json:"api_url"`
-	AuthConfig         AuthConfig              `yaml:"auth" bson:"auth" json:"auth" id:"auth"`
-	AWSInstanceRole    string                  `yaml:"aws_instance_role" bson:"aws_instance_role" json:"aws_instance_role"`
-	Banner             string                  `bson:"banner" json:"banner" yaml:"banner"`
-	BannerTheme        BannerTheme             `bson:"banner_theme" json:"banner_theme" yaml:"banner_theme"`
-	Buckets            BucketsConfig           `bson:"buckets" json:"buckets" yaml:"buckets" id:"buckets"`
-	Cedar              CedarConfig             `bson:"cedar" json:"cedar" yaml:"cedar" id:"cedar"`
-	ClientBinariesDir  string                  `yaml:"client_binaries_dir" bson:"client_binaries_dir" json:"client_binaries_dir"`
-	CommitQueue        CommitQueueConfig       `yaml:"commit_queue" bson:"commit_queue" json:"commit_queue" id:"commit_queue"`
-	ConfigDir          string                  `yaml:"configdir" bson:"configdir" json:"configdir"`
-	ContainerPools     ContainerPoolsConfig    `yaml:"container_pools" bson:"container_pools" json:"container_pools" id:"container_pools"`
-	Credentials        map[string]string       `yaml:"credentials" bson:"credentials" json:"credentials"`
-	CredentialsNew     util.KeyValuePairSlice  `yaml:"credentials_new" bson:"credentials_new" json:"credentials_new"`
-	Database           DBSettings              `yaml:"database" json:"database" bson:"database"`
-	DomainName         string                  `yaml:"domain_name" bson:"domain_name" json:"domain_name"`
-	Expansions         map[string]string       `yaml:"expansions" bson:"expansions" json:"expansions"`
-	ExpansionsNew      util.KeyValuePairSlice  `yaml:"expansions_new" bson:"expansions_new" json:"expansions_new"`
-	GithubPRCreatorOrg string                  `yaml:"github_pr_creator_org" bson:"github_pr_creator_org" json:"github_pr_creator_org"`
-	GithubOrgs         []string                `yaml:"github_orgs" bson:"github_orgs" json:"github_orgs"`
-	DisabledGQLQueries []string                `yaml:"disabled_gql_queries" bson:"disabled_gql_queries" json:"disabled_gql_queries"`
-	HostInit           HostInitConfig          `yaml:"hostinit" bson:"hostinit" json:"hostinit" id:"hostinit"`
-	HostJasper         HostJasperConfig        `yaml:"host_jasper" bson:"host_jasper" json:"host_jasper" id:"host_jasper"`
-	Jira               JiraConfig              `yaml:"jira" bson:"jira" json:"jira" id:"jira"`
-	JIRANotifications  JIRANotificationsConfig `yaml:"jira_notifications" json:"jira_notifications" bson:"jira_notifications" id:"jira_notifications"`
-	Keys               map[string]string       `yaml:"keys" bson:"keys" json:"keys"`
-	// TODO (DEVPROD-4798): remove KanopySSHKeyPath after the cutover to Kanopy.
```

Comment:
do we not want to remove KanopySSHKeyPath anymore, re this comment?

Reply:
The approach of this PR is to remove the old `Keys` map instead of removing the new `KanopySSHKeyPath`. The thinking is that the distros all use the same key (they always have and will continue to use the same key for the foreseeable future), so the `Keys` map and corresponding distro setting was just extra complexity.

We could potentially rename `KanopySSHKeyPath` to `SSHKeyPath` if we want to.